activejob/lib/active_job/serializers/time_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  module Serializers
+    class TimeSerializer < ObjectSerializer # :nodoc:
+      def serialize(time)
+        super("value" => time.to_s)
```

Comment:
`time.iso8601` here and `Time.iso8601(hash["value"])` to deserialize?

Reply:
yeah, it make sense to use a iso format.